app/indexers/ams/work_indexer.rb

```diff
@@ -0,0 +1,25 @@
+module AMS
+  class WorkIndexer < Hyrax::WorkIndexer
+    def generate_solr_document
+      find_index_child_attributes(super)
+    end
+    private
+      def find_index_child_attributes(solr_doc)
+        if solr_doc["member_ids_ssim"]
+          child_works = solr_doc["member_ids_ssim"]
+          child_works.each do |child_id|
+            work = ActiveFedora::Base.search_by_id(child_id)
+              work_type= work[:has_model_ssim].first.constantize
+              parent_indexable_properties = work_type.properties.select{|index,val| val["index_to_parent"]||index=="language"?true:false}
```

Comment:
@afred @jasoncorum right now, i did not find anyway to add index_to_parent option to Hyrax::BasicMetadata properties so hard coding language for now.